detoks/src/main/java/nl/tudelft/trustchain/detoks/TorrentFragment.kt

```diff
@@ -0,0 +1,104 @@
+package nl.tudelft.trustchain.detoks
+
+import android.os.Bundle
+import android.view.View
+import android.widget.*
+import com.frostwire.jlibtorrent.FileStorage
+import com.frostwire.jlibtorrent.TorrentHandle
+import kotlinx.android.synthetic.main.fragment_torrent.*
+import nl.tudelft.ipv8.android.IPv8Android
+import nl.tudelft.trustchain.common.ui.BaseFragment
+
+
+class TorrentFragment : BaseFragment(R.layout.fragment_torrent) {
+
+    private lateinit var torrentManager: TorrentManager
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        torrentManager = TorrentManager.getInstance(requireActivity())
+
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val infoHashTV = view.findViewById<TextView>(R.id.infoHashTextView)
+        val magnetLinkTV = view.findViewById<TextView>(R.id.magnetLinkTextView)
+        val filesTV = view.findViewById<TextView>(R.id.filesTextView)
+        val downloadedBytesTV = view.findViewById<TextView>(R.id.downloadedBytesTextView)
+        val totalWatchTimeTV = view.findViewById<TextView>(R.id.totalWatchTimeTextView)
+        val hopCountTV = view.findViewById<TextView>(R.id.hopCountTextView)
+        val watchedTV = view.findViewById<TextView>(R.id.watchedTextView)
+        val durationTV = view.findViewById<TextView>(R.id.durationTextView)
+        val likesTV = view.findViewById<TextView>(R.id.likesTextView)
+        val timesSeenTV = view.findViewById<TextView>(R.id.timesSeenTextView)
+        val uploadDateTV = view.findViewById<TextView>(R.id.uploadDateTextView)
+
+        val errorTV = view.findViewById<TextView>(R.id.errorTextView)
+
+        val passedName = "electricsheep-flock-248-7500-9"
```

Comment:
Does it still use only this hardcoded value or dynamically get it from the view?

Reply:
Oups, good catch! Should be dynamic now